src/plugin-systeminfo/operation/systeminfomodel.cpp

```diff
@@ -89,6 +91,16 @@ void SystemInfoModel::setHostName(const QString &hostName)
     hostNameChanged(hostName);
 }
 
+void SystemInfoModel::setEnduserAgreement(const QString &text)
```

Comment:
```suggestion
void SystemInfoModel::setEndUserAgreement(const QString &text)
```

以及想简写的话可以写成 EULA。上面用 `endAgreement` 以及 `EnduserAgreement` 都有点要全不全要简不简的感觉。

Reply:
好了